manual/patterns.md

```diff
@@ -164,6 +164,16 @@ It matches with `find_by_email_and_name(...)`.
 - If value of meta var is a string `foo`, it matches send nodes with exactly same method name
 - If value of meta var is a regexp `/foo/`, it matches send nodes with method name which `=~` the regexp
 
+You can also use `as` syntax with `:symbol:` and so on.
+
+```yaml
+id: migration_references
+pattern:
+  subject: 't.integer(:symbol: as 'column, ...)'
```

Comment:
Do single quotes work? (`'column` would do something bad??)

Reply:
Oops, I guess it does not work. Fixed a1a85cdf119f0ffbd6a988e5cabd01a6221872ed